samri/plotting/maps.py

```diff
@@ -370,7 +421,230 @@ def stat(stat_maps,
+
+	col_plus = norm(treshhold)
+	col_minus = norm(-treshhold)
+
+	col_plus = MYMAP(col_plus)
+	col_minus = MYMAP(col_minus)
+
+	col_plus = mcolors.to_hex([col_plus[0],col_plus[1],col_plus[2]])
+	col_minus = mcolors.to_hex([col_minus[0],col_minus[1],col_minus[2]])
+	script_loc = os.path.join(os.path.dirname(sys.argv[0]),'blender_visualization.py')
+	print(script_loc)
+	cli = ['blender', '-b', '-P', script_loc,'--','-t',template_mesh]
+
+	for path in obj_paths:
+		if not path is None:
+			cli.append('-s')
+			cli.append(path)
+			if "neg_mesh" in path: 
+				cli.append('-c') 
+				cli.append(col_minus)
+			if "pos_mesh" in path:
+				cli.append('-c'),
+				cli.append(col_plus)
+
+	#python script cannot be run directly, need to start blender in background via commandline, then run script.
+	subprocess.run(cli,check=True)
+
+	mesh = plt.imread("/var/tmp/3Dplot.png")
+
+	#assure best fit into existing plot, trim img data matrix
+	dims = np.shape(mesh)
+	mask = mesh == 0
+	bbox = []
+	all_axis = np.arange(mesh.ndim)
+	for kdim in all_axis:
+		nk_dim = np.delete(all_axis, kdim)
+		mask_i = mask.all(axis=tuple(nk_dim))
+		dmask_i = np.diff(mask_i)
+		idx_i = np.nonzero(dmask_i)[0]
+		if len(idx_i) != 2:
+			idx_i = [0, dims[kdim]-2]
+		bbox.append([idx_i[0]+1, idx_i[1]+1])
+	mesh_trimmed = mesh[bbox[0][0] : bbox[0][1] ,bbox[1][0]  : bbox[1][1] , :]
+
+	return mesh_trimmed
+
+
+def _plots_overlay(display,display_3Dplot):
+
+	"""Internal function which overlays the plot from stat() with the 3D plot
+
+	Parameters
+	----------
+
+	display : nilearn.plotting.display.OrthoSlicer2 object
+		figure plot as returned by stat() or nilearn.plotting.plot_img(), containing plots in 2x2 view with empty space for 3D image.
+
+	display_3Dplot : array
+		image array.
+
+	"""
+	#get matplotlib figure from Nilearn.OrthoSlicer2 object
+	fh = display.frame_axes.get_figure()    
+	fh.canvas.draw()
+
+	#Determine correct location to put the plot in relation to existing figure axes
+	box = [max(display.axes['x'].ax.get_position().x0,display.axes['y'].ax.get_position().x0,display.axes['z'].ax.get_position().x0),min(display.axes['x'].ax.get_position().y0,display.axes['y'].ax.get_position().y0,display.axes['z'].ax.get_position().y0),display.axes['x'].ax.get_position().bounds[2],display.axes['z'].ax.get_position().bounds[3]]
+
+	#add new axes
+	ax_mesh = fh.add_axes(box)
+
+	#set all backgtuond options to invisible
+	ax_mesh.get_xaxis().set_visible(False)
+	ax_mesh.get_yaxis().set_visible(False)
+	ax_mesh.patch.set_alpha(0.1)
+	ax_mesh.spines["top"].set_visible(False)
+	ax_mesh.spines["bottom"].set_visible(False)
+	ax_mesh.spines["right"].set_visible(False)
+	ax_mesh.spines["left"].set_visible(False)
+
+	plt.gca()
+	img_mesh = plt.imshow(display_3Dplot)
+	plt.show()
+	return fh
+
+def stat3D(stat_maps,
+	overlays=[],
+	figure_title="",
+	interpolation="hermite",
+	template="~/usr/shared/mouse-brain-atlases/ambmc2dsurqec_15micron_cut_mesh_1_decimated.obj",
```

Comment:
This is not under the user home dir. This is a system-wide directory, also it is called `/usr/share`, without the “d”.